packages/toolboxes/src/sui/toolbox.ts

```
@@ -60,8 +61,8 @@ export async function getSuiToolbox({ provider: providerParam, ...signerParams }
       }
 
       return suiBalances;
-    } catch (error) {
-      throw new SwapKitError("toolbox_sui_balance_error" as any, { error });
```

why dont you want to throw an error here?